### ISSUE TITLE
Tablet/Portrait mode to display multiple columns at once

### ISSUE THREAD

**Issue by l-mb** [stefan-niedermann/nextcloud-deck#781]
I also use Nextcloud Deck on my tablet (or my phone in landscape mode).

There's enough space to show multiple columns/lists (splitting the screen horizontally, perhaps with configurable width/number of columns?), which would make both visualization as well as interaction more powerful.

It'd be sufficient if all lists scrolled at the same time, but if columns could be scrolled individually as well, that'd be great.

**Comment by stefan-niedermann** [comment]
Yep, that would be an awesome enhancement, i agree. I once had a working prototype for that, back then, when we used `ViewPager` to display the lists.
Unfortunately this doesn't work anymore with `ViewPager2` (which we use for a few months now) and i have no clue how this could be implemented 🤷‍♂️ 

So **yes** to the request, but i need definitely help from the community with ideas about how to implement it.

**Comment by Nico7as** [comment]
Hi! 

A working multi-column view would be awesome.

Any progress in the last 2,5 years?

With Google selling a PixelTab, Android might get some love for tablet-specific improvements!

Keep it up! 
Cheers!
